web/firefoxcom.js

```diff
@@ -324,9 +324,10 @@ class MLManager {
     return FirefoxCom.requestAsync("mlGuess", data);
   }
 
-  enable({ enableAltText, listenToProgress }) {
+  enable({ altTextLearnMoreUrl, enableAltText, listenToProgress }) {
     if (enableAltText) {
       this.#loadAltTextEngine(listenToProgress);
+      this.altTextLearnMoreUrl = altTextLearnMoreUrl;
```

Comment:
Is the intention that the link-element should only present and clickable when altText generation is enabled, or should this property perhaps be set regardless of the `enableAltText` state?

Looking at the Figma-spec, the "Learn more" link at least *looks* clickable regardless of the state of "Create alt text automatically" toggle.